src/libraries/System.Private.CoreLib/src/System/IO/Path.cs

```diff
@@ -816,11 +816,7 @@ private static unsafe string JoinInternal(ReadOnlySpan<char> first, ReadOnlySpan
             }
         }
 
-        private static ReadOnlySpan<byte> Base32Char => new byte[32] { // uses C# compiler's optimization for static byte[] data
-                (byte)'a', (byte)'b', (byte)'c', (byte)'d', (byte)'e', (byte)'f', (byte)'g', (byte)'h',
-                (byte)'i', (byte)'j', (byte)'k', (byte)'l', (byte)'m', (byte)'n', (byte)'o', (byte)'p',
-                (byte)'q', (byte)'r', (byte)'s', (byte)'t', (byte)'u', (byte)'v', (byte)'w', (byte)'x',
-                (byte)'y', (byte)'z', (byte)'0', (byte)'1', (byte)'2', (byte)'3', (byte)'4', (byte)'5' };
+        private static ReadOnlySpan<byte> Base32Char => "abcdefghijklmnopqrstuvwxyz012345"u8;
```

Comment:
ended up looking at Populate83FileNameFromRandomBytes. do you know why it's taking bits of the random bytes in such a peculiarly special way? why doesn't it just take as many bytes as it needs, skip any beyond a multiple of the length of this array, and just modulo over this array? perhaps because random bytes gotten this way are expensive?

Reply:
it's not obvious to me that the way it's doing right now doesn't cause bias.